src/app/wizard/step/cluster/style.scss

```diff
@@ -22,6 +22,10 @@ $toggle-height: 45px;
   margin: 4px 8px;
 }
 
+.km-label-primary {
```

Comment:
One last thing is to check if it can be removed and if you can use `fxLayoutGap="3px"` in the template instead.

Reply:
I did try that but it doesn't work. Because `margin` is inherited from `km-label-primary` and it ruins the layout.